src/components/Form/index.tsx

```diff
@@ -100,23 +100,17 @@ export const Form = ({
   const inputFieldLabelArr = [
```

Comment:
is there a need for this object ?
if not delete it

Reply:
oh , we need them .... ok just format the code

Reply:
no, we need it but I delete un used code